### PR TITLE
manifest: pull Matter Wi-Fi network diagnostics implementation

### PR DESCRIPTION
**Pull request by markaj-nordic**
This change adds support the the wifinetworkdiagnostics cluster.
Following statistics can be read:
  * multicast RX/TX
  * unicast RX/TX
  * beacons lost/TX

Signed-off-by: Marcin Kajor <marcin.kajor@nordicsemi.no>

### CODE REVIEW

**Review by LuDuda — COMMENTED** [review]
Please add the information in release notes about this addition. Otherwise, looks good!

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -69,7 +69,9 @@ Bluetooth mesh
 Matter
 ------
 
-|no_changes_yet_note|
+* Added:
+
+  * Support for Wi-Fi Network Diagnostic Custer (the counts for the number of packets received and transmitted on the Wi-Fi interface)
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:74]:
```suggestion
  * Support for Wi-Fi Network Diagnostic Cluster (which counts the number of packets received and transmitted on the Wi-Fi interface).
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
I guess we need to update sha in the links.txt for all the Matter docset links, right, @Damian-Nordic ?

**Comment by kkasperczyk-no** [comment]
> I guess we need to update sha in the links.txt for all the Matter docset links, right, @Damian-Nordic ?

@greg-fer I think we don't need to do that. This change updates only code and nothing related to docs didn't change, so it should be fine.

**Review by kkasperczyk-no — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**